hdfs3/core.py

```diff
@@ -296,14 +296,22 @@ def glob(self, path):
         out = [f for f in allfiles if fnmatch.fnmatch(ensure_string(f), path)]
         return out
 
-    def ls(self, path):
+    def ls(self, path, l=True):
+        """ List files at path
+
+        if l==True (default), each list item is a dict of file properties;
+        otherwise, returns list of filenames
```

Comment:
Suggest using `Parameters` as follows

```
Parameters
---------
path: string
    Path on HDFS
all: bool, optional
    If True then each list item is a dict of file properties; otherwise returns a list of filenames
```

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections